exp/capacity-planning.md

```diff
@@ -0,0 +1,96 @@
+When planning your Anbox Cloud deployment, you should start by estimating how much capacity you need to be able to provide your application to your users and how many users you expect. Based on this estimate, you can then size your deployment and figure out how many cluster nodes you need and what resources they should have.
+
+In your calculation, you should consider the following aspects:
+
+- Application resources - How much CPU, memory and disk space does your application need? Will the application use hardware- or software-based video encoding, and if it uses hardware-based encoding, how much GPU capacity is needed?
+- Over-commitment - Does every container need dedicated access to the CPU and memory, or can the capacity be shared between several containers?
+- Application requirements - What type of application are you running? What frame rate and what resolution does your application need? How many containers will be running simultaneously? What would be the impact of not being able to serve all users?
+
+See the following sections for detailed information on these aspects.
+
+## Application resources
+
+Depending on the resources that your application requires, choose a suitable [instance type](https://discourse.ubuntu.com/t/instance-types/17764).
+
+The instance type specifies the resources that are available to the application:
+
+- The number of vCPU cores
+- The amount of RAM
+- The amount of disk space
+- The number of GPU slots
+
+If none of the provided instance types fits for your application, you can also manually [configure the resources](https://discourse.ubuntu.com/t/how-to-configure-available-resources/24960) according to your requirements.
+
+When a container for an application is launched, it takes the specified amount of resources. For example, if an application uses the `a2.3` instance type, it requires 2 CPU cores, 3 GB of memory, 3 GB of disk space and no GPU slot. AMS internally summarises the amount of resources used by containers on a single machine and disallows launching additional containers when all resources are used (see [Over-committing resources](#overcommitting) for how to allow a higher resource usage).
```

Comment:
If we say "2 CPU cores" we mean vCPU cores. A single vCPU core maps to a time budget on all CPU cores the socket has available. If we want to keep things simple, lets us say `... it requires 2 vCPU cores`